src/stores/account.js

```diff
@@ -29,6 +29,9 @@ export const useAccountStore = defineStore({
     accountData(state) {
       return state.account
     },
+    userFlags(state) {
```

Comment:
As flags são adicionadas para toda a conta. Não por usuário. Vamos alterar para
```suggestion
    clientFlags(state) {
```
?